cleanlab/token_classification/filter.py

```diff
@@ -1,22 +1,24 @@
 import numpy as np
 from cleanlab.filter import find_label_issues as find_label_issues_main
-from typing import Optional, Union, Tuple
+from typing import List, Tuple
 
 
 def find_label_issues(
-    labels: list, pred_probs: list, return_indices_ranked_by: str = "self_confidence"
-) -> list:
+    labels: List[List[int]],
+    pred_probs: List[np.ndarray],  # type: ignore[type-arg]
```

Comment:
`type: ignore[type-arg]` is unnecessary for our current mypy configuration. And when we do switch to strict mode, I think we might want some other way of handling this, e.g., explicit `np.ndarray[Any]` everywhere we allow any type.

Reply:
Removing ignore-comment.